jdxsmith/whats-cookin
# Refactor/html css

#### What does this PR do?
Finalizes HTML and CSS styling by:
- Fixing pantry banner to not run off the page
- Adding a remove button to the recipe cards displayed in favorites/recipes to cook
- Adding a reset search button to the top of the page when tags/ingredients are searched
- Ensures inactive buttons are correctly displayed across page views
- Fixes text display on missing ingredients and no recipes to display
- Gets rid of commas that were displaying on ingredient and instruction lists
- Fixes bug that displayed cost as more than 2 decimal places

#### Where should the reviewer start?
`index.html`, `styles.css` and `scripts.js`.

#### How should this be manually tested?
1. Search by tag and ingredient on main, favorites and recipes to cook page and test the 'reset search' button
2. Remove a recipe from favorites and recipes to cook
3. Switch between page views to ensure the correct page is displaying
4. Open up several recipe card to ensure the cost is only displaying 2 decimals and the commas are not displaying on ingredients or instructions
5. Resize the pantry page to ensure the banner stays overlapping the image

#### Any background context you want to provide?
I want to go back and comb through CSS to ensure all styles are being used and that the order flows with the page (as possible)
